command/operator/usage/instances/usage_instances.go

```diff
@@ -135,9 +137,11 @@ func formatNodesCounts(usageStats map[string]api.ServiceUsage) (string, error) {
 
 	fmt.Fprint(tw, "\t\n")
 
-	for dc, usage := range usageStats {
-		nodesTotal += usage.Nodes
-		fmt.Fprintf(tw, "%s\t%d\n", dc, usage.Nodes)
```

Comment:
Tests with dc1 and dc2 were expecting them to be printed in order. Maps are not iterated deterministically so we want to sort the datacenters before iterating.